lib/rubocop/cop/style/access_modifier_declarations.rb

```diff
@@ -129,10 +129,17 @@ def allow_modifiers_on_symbols?(node)
         end
 
         def offense?(node)
-          (group_style? && access_modifier_is_inlined?(node)) ||
+          (group_style? && access_modifier_is_inlined?(node) &&
+            !right_siblings_same_inline_method?(node)) ||
             (inline_style? && access_modifier_is_not_inlined?(node))
         end
 
+        def right_siblings_same_inline_method?(node)
```

Comment:
That's just my two cents. Can you move this new method to under the `access_modifier_is_not_inlined` method? (i.e. around L158)

Reply:
Thank you. I updated this PR.